manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml

```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: "WondershareSoftwareCoLtd.WondershareFilmora"
+PackageVersion: "10.5"
+PackageLocale: "en-US"
+Publisher: "Wondershare Software Co. Ltd."
```

Comment:
```suggestion
Publisher: Wondershare Software
```
ARP

Reply:
no

Reply:
publisher is correct, as seen in official website

Reply:
> publisher is correct, as seen in official website

![image](https://user-images.githubusercontent.com/15158490/124385044-61633080-dcd4-11eb-9f8a-74e72e1a24a5.png)

Its not.

Reply:
Thanks @OfficialEsco! The PackageName, Publisher and the PackageVersion must be the same as in the uninstall list. (ARP)

Reply:
sorry, here is has been written "Wondershare Software Co. Ltd." [in pdfelement installation]
![image](https://user-images.githubusercontent.com/83997633/124388524-fd0a9600-dd00-11eb-9146-f9e986a505eb.png)

Reply:
changes made...